src/components/App/App.js

```diff
@@ -1,11 +1,21 @@
 import React, { Component } from 'react';
 import queries from '../../utils/queries/queries';
 import LoginForm from '../LoginForm/LoginForm';
+import GuardianDetailsForm from '../GuardianDetailsForm/GuardianDetailsForm';
 
 class App extends Component {
   state = {
     error: '',
-    users: [] 
+    users: [],
```

Comment:
Are we adding these props just to test it out? I added the Controls form to App so the routes should be working once I make a PR